main.cpp

```diff
@@ -78,6 +174,30 @@ int main(int argc,char *argv[])
 	limit.rlim_cur = 1024;
 	limit.rlim_max = 1024;
 
+	// Browse argv and try to find the --aclFile option
+	int found_acl = 0;
+	int k = 0;
+	while(k<argc && !found_acl) {
```

Comment:
Should be **for**, not **while**

Reply:
Our first and most important rule is consistency: if there isn't a very good reason to change keep the things how we find. And this is what I have done with this patch. In other words in our opinion or we accept this form (we also follow already present indentation!) or we change all code which have already this form. This applies also on all other your comments: -maxFile and -poolSize for argument names, for use of the while **loop** instead of the **for** one, for extraction a method from a common few lines of code and for the name of the binary.

Reply:
I tend to agree with Alessio here for this PR (consistency is important) but we should keep in mind that this code could be improved indeed as suggested by Igor.
What I would suggest is to keep the code consistent for the next TANGO 9 releases. So I would even rename the executable to the name it was in Tango 9.2.5a official release. I would not rename an executable in a minor release.
For TANGO 10, we are more free and could indeed refactor the code, options names and executable name to follow the most followed standards.
> The nice thing about standards is that there are so many of them to choose from - Andrew S. Tanenbaum